src/main/java/com/scalyr/integrations/kafka/AddEventsClient.java

```diff
@@ -92,6 +93,20 @@
    */
   private static final AddEventsResponse TIMEOUT_ADD_EVENTS_RESPONSE = new AddEventsResponse().setStatus("Timeout").setMessage("Timeout");
 
+  /** Add events API has max payload size of 6 MB */
+  private static final int MAX_ADD_EVENTS_PAYLOAD_BYTES = 6000000;
```

Comment:
I would still set this a little under 6MB.  It could be that we should taking headers into a consideration, etc.  So, let's do 5_500_000.

Reply:
We are skipping add event calls that go over this size.  If we set this lower than 6 MB, we could skip additional add event requests that would be accepted on the server.  Maybe we change the BATCH_SEND_SIZE_BYTES_CONFIG validation for a max of 5_500_000?

Reply:
Sure... and it's fine to reject a few batches erroneously if it gets us in a better position, IMO.